### DISCUSSION TITLE
ESP32 DeepSleep: Can someone confirm that the device is really waking up again?

### DISCUSSION THREAD

**Discussion by h0lad** [meshtastic/Discussions#98] [Firmware]
Hey All!

I have the feeling that there might be something broken with DeepSleep and Low Voltage.

Can somebody test

1. Reduce DeepSleep  Delay to a reasonably testable value
2. Let it go into DeepSleep by Low Voltage
3. Set the Voltage back to normal
4. Check if it is waking up again

Does this work?

I expected one of my nodes from home to wake up after a bad weather period (solar). But it still seems to be dead. 'sds' was set to 6h.

Cannot check if myself right now as I'm traveling. 

Thank you guys/girls!

**Comment by mariotti** [comment]
Hoi, the fact that you have no time, might not be a real nice reason why I need to spend my time reading your issues.

This given, I am not sure what you need to test.

It looks like that you want to test:

If the device has almost no power, would it wake up when the power gets back to normal?

I'd say, first you might need to check if the device, indeed shutdown, due to low power.
In that case, I think the problem is solved: "no batteries".

Then each device might behave differently in low power or no power condition (here all the same indeed).
But definitely how they wake up once power is restored.

So we are also missing details.

I suggest to close this discussion.

Best
A+
F

**Reply by h0lad** [reply]
> I suggest to close this discussion.

I'm sorry, I'm not going to comply with this request.

I will have access to my lab again in a week at the latest and will then be able to test it myself. 

Since I have found some critical problems in the environment of low bat behavior of Meshtastic before, I don't want to rule this out completely that there is a problem around the wake up/deep sleep functionality. It's a gut feeling.

This might be specialty because the ESP32C3 on the Heltec HT-CT62 is being used. This variant has a problem with the serial port that a flush might cause issues with deep sleep.

**Comment by h0lad** [comment]
Maybe related: https://forum.seeedstudio.com/t/esp32c3-will-not-wake-up-after-deepsleep-time/269513/5
